_posts/2021-01-06-how-to-update-to-cis-13.adoc

```diff
@@ -0,0 +1,232 @@
+incompatible releases for any version updates before the 1.0.0 release. Make sure to read the release notes for the
+relevant modules any time you are updating minor versions! Note that you will want to read the release notes for each
+minor version that is updated (e.g., if you are going from v0.5.x to v0.9.x, you will want to read the notes for v0.6.0,
+v0.7.0, v0.8.0, and v0.9.0 to get the full list of backward incompatible updates).
+
+The following table provides a summary of all the relevant Gruntwork AWS modules and the respective versions that are
+compatible with CIS AWS v1.3.0:
+
+[[compatibility_table]]
+[cols="1,1h,1"]
+|===
+|Gruntwork Repo |Minimum version with CIS AWS v1.3.0 support |Corresponding CIS AWS v1.3.0 recommendations
+
+|terraform-aws-security
+|https://github.com/gruntwork-io/terraform-aws-security/releases/tag/v0.44.6[v0.44.6]
+|1.20, 1.21, 2.1.1, 2.1.2, 3.10, 3.11
+
+|terraform-aws-monitoring
+|https://github.com/gruntwork-io/terraform-aws-monitoring/releases/tag/v0.24.0[v0.24.0]
+|1.20, 2.1.1, 2.1.2
+
+|terraform-aws-zookeeper
+|https://github.com/gruntwork-io/terraform-aws-zookeeper/releases/tag/v0.8.0[v0.8.0]
+|1.20, 2.1.1, 2.1.2
+
+|terraform-aws-vpc
+|https://github.com/gruntwork-io/terraform-aws-vpc/releases/tag/v0.13.0[v0.13.0]
+|1.20, 2.1.1, 2.1.2
+
+|terraform-aws-openvpn
+|https://github.com/gruntwork-io/terraform-aws-openvpn/releases/tag/v0.13.0[v0.13.0]
+|1.20, 2.1.1, 2.1.2
+
+|terraform-aws-cis-service-catalog
+|https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/releases/tag/v0.10.0[v0.10.0]
+|3.10, 3.11, 4.15, 5.1
+
+|terraform-aws-service-catalog
+|https://github.com/gruntwork-io/terraform-aws-service-catalog/releases/tag/v0.15.4[v0.15.4]
+|1.21
+
+|===
+
+
+=== Step 2: Manual steps
+
+Recommendation 4.15 requires an active subscriber for the SNS topic created by this recommendation (see Audit steps 6
+and 7 for 4.15). As it's impossible to automate this subscriber's creation, you'll have to do so manually. See
+https://docs.aws.amazon.com/sns/latest/dg/sns-create-subscribe-endpoint-to-topic.html[Subscribing to an Amazon SNS topic]
+on the AWS website for detailed instructions.
+
+
+=== Step 3: Deploy new modules
+
+==== 1. Deploy the Cleanup Expired Certs module (recommendation 1.19)
+
+The new CIS AWS v1.3 recommendations require that all expired SSL/TLS certificates stored in AWS IAM are automatically removed. Removing expired SSL/TLS certificates eliminates the risk that an invalid certificate will be deployed
+accidentally to a resource such as AWS Elastic Load Balancer (ELB), which can damage the credibility of the application/website behind the ELB. As a best practice, it is recommended to delete expired certificates. To help you
+achieve this recommendation, check out the https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/tree/master/examples/cleanup-expired-certs/terraform[example] of the ready-made `cleanup-expired-certs` module.
```

Comment:
You can also cross-link to the 1.3.0 guide, as it now has a section on how to use this module (with Terragrunt), and similar sections for each module below!

Reply:
Added in 8ada2d539f0d5988ff84905101214d750accfb40. (Note that these links don't resolve right now because the 1.3.0 guide hasn't been merged yet!)

Reply:
Added in 8ada2d539f0d5988ff84905101214d750accfb40. (Note that these links don't resolve right now because the 1.3.0 guide hasn't been merged yet!)